CONTRIBUTING.md

```diff
@@ -53,6 +53,23 @@ When you're ready to contribute, follow these steps:
 
 **Note:** for some special cases, you'd need to contact us to create a branch in this repository (not fork). See below.
 
+### Writing Tickets and PR Descriptions
+
+When writing feature tickets (or opening PRs without an attached ticket), please ensure that the following items are included in your ticket or PR. Always assume that the reviewer may not know exactly what you are working on or what your motivations are. This should be clear from the ticket or PR comment. When in doubt, feel free to use ChatGPT to help formulate clear descriptions :) You can also check our GitHub ticket templates.
```

Comment:
this is all part of a issue template. we do not need to repeat that. or am I missing something?